lib/ohai/plugins/vmware.rb

```diff
@@ -0,0 +1,77 @@
+#
+# Author:: "Dan Robinson" <drobinson@getchef.com>
+# Author:: "Christopher M. Luciano" <cmlucian@us.ibm.com>
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+# Copyright (C) 2015 IBM Corp.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Provides a set of attributes for a VMware ESX virtual machine with results
+# obtained from vmware-toolbox-cmd.  VMware Tools must be installed
+# on the virtual machine.
+#
+# Modify the path to vmware-toolbox-cmd in the call to get_vm_attributes for
+# your particular operating system and configuration
+#
+# Example:
+#
+# get_vm_attributes("/usr/bin/vmware-toolbox-cmd")
+#
+
+Ohai.plugin(:Vmware) do
```

Comment:
(Nitpick warning): Ohai plugin names are case sensitive. Should we name this `:VMware` to match the company/software name?